tutorials/compute-managed-instance-groups-dashboard/index.md

```diff
@@ -0,0 +1,182 @@
+Simple MIG Dashboard
+====================
+
+Introduction
+------------
+
+Simple MIG dashboard is a tool for monitoring the state of instances of
+a Managed Instance Group. For each instance it visualizes its current
+action (e.g. *CREATING*, *RESTARTING*) and its Instance Template. It can be
+especially useful to monitor progress of a rolling update: it shows how
+the state (including health) of each instance is changing over time.
+
+This tutorial explains how to set up and use Simple MIG dashboard on
+your local machine with your Google Cloud Platform project. It walks you
+through the process of setting up a GCP web application and granting it
+all necessary permissions to access your project data.
+
+Simple MIG dashboard is implemented in JavaScript using Angular
+framework. It uses Google Charts library for visualization purposes and
+Google API Client Library for JavaScript to communicate with Google
+Cloud Platform.
+
+Before you begin
+----------------
+
+In this tutorial, we assume that you have a Google Cloud Platform
+project with an existing Managed Instance Group, which you want to
+monitor.
+
+Running the dashboard
+---------------------
+
+To run the dashboard locally, you need to set up a new GCP application.
+Follow the steps
+[here](https://support.google.com/cloud/answer/6158849?hl=en&ref_topic=6262490)
+to obtain Client ID, that will be used to identify your application when
+making API calls. Choose *Web application* as application type, and add
+[http://localhost:8080](http://localhost:8000) to the list
+of Authorized JavaScript origins for your app.
+
+Once you obtained Client ID for your application, you are ready to run
+the dashboard locally:
+
+1.  `git clone https://github.com/GoogleCloudPlatform/community`
+
+2.  `cd community/tutorials/compute-managed-instance-groups-dashboard/webapp/`
+
+3.  Edit `gapi.js` file: replace *clientId* in line 44 with your Client ID
+
+4.  From `webapp/` run: `python -m SimpleHTTPServer 8080`
+
+Now go to [http://localhost:8080](http://localhost:8080)
+in your browser, and you'll be greeted with a window which allows you to
+log in to your Google account. Choose the account which has the access
+to your GCP project. In the next window, Google will ask for your
```

Comment:
change: "In the next window, Google will ask for your"
to
"The next window asks for your..."